### ISSUE TITLE
Feature Request: option for manual unpause

### ISSUE THREAD

**Issue by 0stiarius** [kyleneideck/BackgroundMusic#322]
I do call center work from home and BM works beautifully to pause music when a call comes in and I get an alert that goes to my headphone jack. When the call begins, the sound is routed to a USB headphone and BM automatically unpauses the music during the call. :-( 

Some way to prevent automatic unpause and require manual unpause would be very helpful.

**Comment by kyleneideck** [comment]
Good idea. I'm thinking we could add an item to the Preferences menu called "More preferences..." or "Advanced..." or something like that. It would open a window and that would have a checkbox for this setting.

Actually, #203 would probably cover this as well. If you could set the delay for pausing and unpausing, setting the unpause delay to 9999999 would have the same effect. I think it's still worth having the checkbox, though, just to make it obvious.

**Comment by endryisnotcool** [comment]
Hello! I would love to contribute to this problem! Mind if I give it a try?
